src/pages/patientView/PatientViewPage.tsx

```diff
@@ -307,21 +307,24 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                             />
 
                             {
-                                (patientViewPageStore.mutationData.isComplete && patientViewPageStore.cnaSegments.isComplete && sampleManager) && (
-                                    <GenomicOverview
-                                        mergedMutations={patientViewPageStore.mergedMutationData}
-                                        cnaSegments={patientViewPageStore.cnaSegments.result}
-                                        sampleOrder={sampleManager.sampleIndex}
-                                        sampleLabels={sampleManager.sampleLabels}
-                                        sampleColors={sampleManager.sampleColors}
-                                        sampleManager={sampleManager}
-                                        getContainerWidth={()=>$(window).width()}
-                                    />
+                                (patientViewPageStore.mutationData.isComplete && patientViewPageStore.cnaSegments.isComplete && sampleManager)
+                                && ( patientViewPageStore.mutationData.result.length > 0 || patientViewPageStore.cnaSegments.result.length > 0)
```

Comment:
may not be necessary but could you wrap each of these predicates in parentheses?